dialogue-target/build.gradle

```diff
@@ -17,5 +17,6 @@ dependencies {
     testCompile 'org.mockito:mockito-core'
 
     annotationProcessor 'org.immutables:value'
+    compileOnly 'org.immutables:value::annotations'
```

Comment:
@iamdanfox AFAIK I do need a `compile` dependency here, otherwise projects consuming this library won't have the `@Generated` annotation on the classpath. Right?